Cargo.toml

```diff
@@ -45,3 +45,6 @@ incremental = false
 
 [workspace.metadata.release]
 consolidate-commits = true
+
+[workspace.dependencies]
+libp2p = { version = "0.50" }
```

Comment:
can you remove this from this PR here please, see discussion on #595

Reply:
Done